docs/ja/docs/deployment/deta.md

```diff
@@ -0,0 +1,240 @@
+# Deta にデプロイ
+
+このセクションでは、**FastAPI** アプリケーションを <a href="https://www.deta.sh/?ref=fastapi" class="external-link" target="_blank">Deta</a> に簡単にデプロイして無料プランを利用する方法を学習します。🎁
```

Comment:
```suggestion
このセクションでは、**FastAPI** アプリケーションを <a href="https://www.deta.sh/?ref=fastapi" class="external-link" target="_blank">Deta</a> の無料プランを利用して、簡単にデプロイする方法を学習します。🎁
```

This might be more natural.